tools/packages/autoconf/autoconf.url

```diff
@@ -1 +1 @@
-ftp://ftp.gnu.org/pub/gnu/autoconf/autoconf-2.68.tar.gz
```

Comment:
Awesome!  I believe HTTPS is supported as well, it might be far more prudent to move to that scheme.

Reply:
Unfortunately, HTTPS, while a supported transport isn't guaranteed to have a supported certificate chain:

```
% wget https://ftp.gnu.org/pub/gnu/make/make-4.1.tar.gz
--2019-07-16 16:34:03--  https://ftp.gnu.org/pub/gnu/make/make-4.1.tar.gz
Resolving ftp.gnu.org... 2001:470:142:3::b, 209.51.188.20
Connecting to ftp.gnu.org|2001:470:142:3::b|:443... connected.
ERROR: cannot verify ftp.gnu.org's certificate, issued by ‘CN=Let's Encrypt Authority X3,O=Let's Encrypt,C=US’:
  Unable to locally verify the issuer's authority.
To connect to ftp.gnu.org insecurely, use `--no-check-certificate'.
% which wget
/usr/local/gnu/bin/wget
```

Depending on where curl or wget was built and installed.